app/ts-store/transport/handler/processor_test.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2022 Huawei Cloud Computing Technologies Co., Ltd.
```

Comment:
```suggestion
Copyright 2024 Huawei Cloud Computing Technologies Co., Ltd.
```

Reply:
This is a common problem and can be solved as a special item.